modules/update-upgrading-web.adoc

```diff
@@ -62,6 +62,12 @@ and click *Save*.
 +
 The Input channel
 *Update status* changes to *Update to <product-version> in progress*, and you can review the progress of the cluster update by watching the progress bars for the Operators and nodes.
++
+[NOTE]
+====
+If you are upgrading your cluster to the next minor version, like version 4.y to 4.(y+1), it is recommended to confirm your nodes are upgraded before deploying workloads that rely on a new feature. Any pools with worker nodes that are not yet updated are displayed on the *Cluster Settings* page. The CVO marks your upgrade as complete and does not wait for the worker nodes to update to the latest version. 
```

Comment:
> The CVO marks your upgrade as complete and does not wait for the worker nodes to update to the latest version.

I would prefer we exclude this sentence.  The cluster-version operator will wait during the update until the machine-config operator [claims to have updated in its ClusterOperator][1].  Since forever, the machine-config operator will wait on compute pools if they include [the `operator.machineconfiguration.openshift.io/required-for-upgrade` label][2].  Compute pools don't get that label out of the box, but local cluster admins are free to add it.  In addition, since 4.7, the machine-config operator will refuse to level as long as there are degraded pools.  For pools with slow-to-drain, PDB-guarded workloads, that will often block machine-config leveling until the compute pools have completed their updates.

I think the fact that we're calling out "hey, you should check your node versions" here is sufficient, without us needing to dip into those details.  While I like going generic and skipping details, I don't like making claims that are not always true, and claiming that the CVO will not wait for your compute pools is often, but not always, true.

[1]: https://github.com/openshift/enhancements/blob/5ee2f5dc41c7f4ff18f591a30f104ebfcdfa29b9/dev-guide/cluster-version-operator/user/reconciliation.md#clusteroperator
[2]: https://github.com/openshift/machine-config-operator/blob/4ca373b691b6fdb8cfeec42947bb5eee1b5087a2/pkg/operator/sync.go#L38

Reply:
@wking I am good with removing it. If you could leave a note in the Bugzilla for the reporter who has requested this sentence to be added.